### DISCUSSION TITLE
[Feature] Batch Album Sharing

### DISCUSSION THREAD

**Discussion by clembuck** [immich-app/immich#12568] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello,

I think it would be helpful to have the ability to select multiple albums (from the Albums tab, for instance) and share them with one or more users in a single action, rather than having to share each album individually.

**Use case scenario:**
I want to share albums with both my friends and my family. However, I don't want my friends to see family albums, nor do I want my family to see albums meant for my friends.
From my understanding, I can't designate any of them as a "share partner" because I currently wouldn't have a way to hide specific albums from each group individually.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#8543
